xero_accounting.yaml

```diff
@@ -1228,6 +1228,21 @@ paths:
             type: integer
         - $ref: '#/components/parameters/unitdp'
         - $ref: '#/components/parameters/pageSize'
+        - in: query
+          name: IDs
+          x-snake: ids
+          description: Filter by a comma-separated list of BankTransactionIDs.
```

Comment:
I want to be able to use $ref for this IDs but at the same time to supply a different description to make it clear what time of IDs are expected for each endpoint, it looks like it is [not possible](https://swagger.io/docs/specification/v3_0/using-ref/#considerations) `"Any sibling elements of a $ref are ignored."`